src/admin/pages/Settings.vue

```diff
@@ -49,33 +50,38 @@
                                         :errors="errors"
                                         :toggle-loading-state="toggleLoadingState"
                                     />
-                                </tbody>
-                            </table>
-                            <p class="submit"><input type="submit" name="submit" id="submit" class="button button-primary" value="Save Changes" @click.prevent="saveSettings( settingValues[index], index )"></p>
-                        </form>
-                    </div>
-                </template>
-            </div>
-
-            <div class="loading" v-if="showLoading">
-                <loading></loading>
+                                    </tbody>
+                                </table>
+                                <p class="submit"><input type="submit" name="submit" id="submit" class="button button-primary" value="Save Changes" @click.prevent="saveSettings( settingValues[index], index )"></p>
+                            </form>
+                        </div>
+                    </template>
+                </div>
+
+                <div class="loading" v-if="showLoading">
+                    <loading></loading>
+                </div>
             </div>
         </div>
+
+        <SettingsBanner if=" ! hasPro"></SettingsBanner>
```

Comment:
Please remove the white space before ! eg: `<SettingsBanner if="! hasPro"></SettingsBanner>`